src/platform/site-wide/sass/shame.scss

```diff
@@ -589,3 +589,13 @@ img[data-srcset] {
     width: 100%;
   }
 }
+
+.campaign-landing-page {
+  .stories {
+    .story-teaser {
+      img {
+        max-height: 150px;
```

Comment:
This image is kinda messed up, not sure what the standard is here for putting height/width on images so wanted to ask 🙂